integration/setup/teardown.go

```diff
@@ -9,50 +9,36 @@ import (
 	"github.com/aws/aws-sdk-go/aws"
 	"github.com/aws/aws-sdk-go/service/cloudformation"
 	"github.com/giantswarm/e2e-harness/pkg/framework"
-	awsclient "github.com/giantswarm/e2eclients/aws"
 	"github.com/giantswarm/microerror"
 
 	"github.com/giantswarm/aws-operator/integration/env"
 )
 
```

Comment:
`HostPeerVPC` is renamed to `deleteHostPeerVPC`, moved to the bottom, and called from `teardown`.